src/zcash/JoinSplit.cpp

```diff
@@ -154,10 +110,6 @@ class JoinSplitCircuit : public JoinSplit<NumInputs, NumOutputs> {
         uint64_t vpub_new,
         const uint256& rt
     ) {
-        if (!vk || !vk_precomp) {
-            throw std::runtime_error("JoinSplit verifying key not loaded");
```

Comment:
I'm going to just alter the test to check for the error we actually get now, given that this error has been removed.